### DISCUSSION TITLE
Marking really common words as misspelled

### DISCUSSION THREAD

**Discussion by Tobbe** [streetsidesoftware/vscode-spell-checker#1180] [Q&A] [answered]
Hi, thanks for the extension 🙂 

It seems some very common words are "misspelled". Have I misconfigured something? Or does it really not know the word "about" etc? Here's a small sample of what I'm seeing
![image](https://user-images.githubusercontent.com/30793/130501645-4a1ad9c8-d578-4ee2-aadd-70c042d1196b.png)

It does suggest other words, so *something* is working
![image](https://user-images.githubusercontent.com/30793/130501769-f091c4d8-ed1c-4214-94a9-727d166f94c2.png)

It's a `.js` file and I don't have any extra dictionaries installed. Here's all the extensions I have
![image](https://user-images.githubusercontent.com/30793/130502054-81df1c45-9603-446d-919b-a3d346dc0d47.png)

If those words really aren't in the dictionary, can they be added? If so, where do I send the PR? 🙂

**Comment by Jason3S** [comment, marked as answer]
Hello Todd,

All of those words are in the English dictionaries included with the spell checker. The behavior you are seeing looks like the English dictionary has been turned off. The language dictionaries are controlled with the `cSpell.language` setting.

See:
![image](https://user-images.githubusercontent.com/3740137/130562279-13b06009-fba3-4874-baaa-a5544924fb0c.png)

**Reply by Tobbe** [reply]
It looks like it's configured correctly

![image](https://user-images.githubusercontent.com/30793/130567428-0acba075-f8cc-42c8-bfdb-8ce454d92c26.png)

**Reply by Tobbe** [reply]
Just now saw this:
> *Modified in: _Workspace_*

![image](https://user-images.githubusercontent.com/30793/130567918-961cc2fc-1f69-4db3-bfd6-3e0c4b31e08a.png)

I have a Swedish dictionary installed on another machine. It must have synced that setting over

**Reply by Tobbe** [reply]
Changing the Workspace setting to also say `en` made it work. Thank you!
